neuralmonkey/writers/plain_text_writer.py

```diff
@@ -1,27 +1,33 @@
 from typing import Iterator, List, Any, Callable
 
 from neuralmonkey.logging import log
+from neuralmonkey.readers.plain_text_reader import ALNUM_CHARSET
 
 # pylint: disable=invalid-name
 # Writer: function that gets file and the data
 Writer = Callable[[str, Any], None]
 # pylint: enable=invalid-name
 
 
-def tokenized_text_writer(encoding: str = "utf-8") -> Writer:
-
-    def writer(path: str, data: Iterator[List[str]]) -> None:
-        with open(path, "w", encoding=encoding) as f_out:
-            for sentence in data:
-                f_out.write(" ".join(sentence) + "\n")
-        log("Result saved as tokenized text in '{}'".format(path))
+def t2t_detokenize(data: Iterator[List[str]]) -> Iterator[str]:
+    """Detokenize text tokenized by t2t_tokenized_text_reader.
 
-    return writer
+    Method is inspired by tensor2tensor tokenizer:
+    https://github.com/tensorflow/tensor2tensor/blob/v1.5.5/tensor2tensor/data_generators/text_encoder.py
```

Comment:
když to rozkliknu, tak tam zaboha nemůžu najít, kde přesně se dělá to, co se dělá tady.

Reply:
Opraveno, melto byt tokenizer.py misto text_encoder.py